src/JSInterop/Microsoft.JSInterop.JS/src/src/Microsoft.JSInterop.ts

```diff
@@ -19,7 +19,24 @@ module DotNet {
    * @param dispatcher An object that can dispatch calls from JavaScript to a .NET runtime.
    */
   export function attachDispatcher(dispatcher: DotNetCallDispatcher) {
+    if (!dispatcher) {
+      throw new Error("dispatcher can't be null or undefined.");
```

Comment:
Should be single quotes, but TBH I think we'd be fine without this check. In most places in TypeScript we rely on compile-time strict nullness checking, since runtime checks aren't super robust anyway (just because a value is nonnull doesn't mean it has anything like the shape you're expecting at runtime).

Reply:
I removed the check for this (and the default implementation) I only added it because it was complaining below. That said, it can't be single quotes due to the single quote in the middle of the string. Normally when you have such quote is acceptable to use double quotes to surround the string.

We don't have ESLint setup in this repo, we could if we wanted to, but I'm not planning to do it now.

Reply:
`\'` 😄

Reply:
got rid of it.